squirrel_interaction/src/squirrel_interaction/board/board.py

```diff
@@ -42,11 +43,12 @@ def __init__(self):
             #self.should_close_door = False
         except SerialException:
             rospy.logwarn('Failed to open some devices')
-        rospy.init_node('squirrel_driver', anonymous=False)
-        print 'Started squirrel_driver node'
+        rospy.loginfo('Started squirrel_driver node')
         rospy.Subscriber('/head_controller/joint_group_position_controller/command', Float64, self.move_head)
         rospy.Subscriber('/neck_pan_controller/joint_group_position_controller/command', Float64, self.move_neck)
+        rospy.Subscriber('/neck_pan_controller/joint_group_position_controller/rel_command', Float64, self.move_neck_rel)
```

Comment:
please don't use global namespaces, see https://github.com/squirrel-project/squirrel_driver/pull/140/files.

it is needed to harmonize the simulation and the real driver also it is a best practice method: use relative namespaces and add the prefix using the launch file tags